sites/_defaults.scss

```diff
@@ -141,6 +142,26 @@ form .btn:not(.btn-call-to-action), .form-control {
 legend {
 	line-height: 1.65em;
 }
+.gc-font-2019 {
+	&, & ~ .wb-fieldflow-form, & + div {
```

Comment:
This selector should be ideally component specific. Currently when I apply `gc-font-2019` on the first fieldflow component, it's all components in the page that are transformed. It is acting like a global CSS that we would apply to the body element.